torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1382,16 +1382,13 @@ def five_crop_video(
     return five_crop_image_tensor(video, size)
 
 
+ImageOrVideoTypeJIT = Union[features.ImageTypeJIT, features.VideoTypeJIT]
```

Comment:
Necessary to get around the JIT issue.

Reply:
JIT is just so cumbersome ... It seems it recognizes that this `Union` flattens to `torch.Tensor`, but if we use that for the output it bails out ...